helmfile/charts/networkpolicy-service/templates/certmanager-np.yaml

```diff
@@ -0,0 +1,170 @@
+{{- range $namespace := .Values.certManager.namespaces }}
```

Comment:
Could you split these files into seperate files in a folder as we discussed? And also, rename to `cert-manager` instead of `certmanager`